src/task.h

```diff
@@ -25,9 +25,11 @@ typedef struct task {
 typedef struct {
    timestamp max_runtime_ns;
    void     *max_runtime_func;
-   uint64    total_latency_ns;
-   uint64    total_tasks;
-   uint64    max_latency_ns;
+   uint64    max_queue_wait_time_ns;
```

Comment:
As long as you are renaming these variables, I suggest you consider renaming these as:

- `queue_wait_time_hwm_ns`
- `outstanding_tasks_hwm`

The `hwm` is a commonly understood acronym for High-Water Mark and is usually used to denote such fields that track the 'max-value of something".

Reply:
Never heard of that.  Already discussed this rename in previous convos.